extensions/wikia/DesignSystem/ds.built.js

```diff
@@ -0,0 +1,768 @@
+						alreadyImportedModules[id] = true;
+				}
+				for(i = 0; i < modules.length; i++) {
+					var item = modules[i];
+					// skip already imported module
+					// this implementation is not 100% perfect for weird media query combinations
+					//  when a module is imported multiple times with different media queries.
+					//  I hope this will never occur (Hey this way we have smaller bundles)
+					if(typeof item[0] !== "number" || !alreadyImportedModules[item[0]]) {
+						if(mediaQuery && !item[2]) {
+							item[2] = mediaQuery;
+						} else if(mediaQuery) {
+							item[2] = "(" + item[2] + ") and (" + mediaQuery + ")";
+						}
+						list.push(item);
+					}
+				}
+			};
+			return list;
+		};
+
+		function cssWithMappingToString(item, useSourceMap) {
+			var content = item[1] || '';
+			var cssMapping = item[3];
+			if (!cssMapping) {
+				return content;
+			}
+
+			if (useSourceMap && typeof btoa === 'function') {
+				var sourceMapping = toComment(cssMapping);
+				var sourceURLs = cssMapping.sources.map(function (source) {
+					return '/*# sourceURL=' + cssMapping.sourceRoot + source + ' */'
+				});
+
+				return [content].concat(sourceURLs).concat([sourceMapping]).join('\n');
+			}
+
+			return [content].join('\n');
+		}
+
+		// Adapted from convert-source-map (MIT)
+		function toComment(sourceMap) {
+			// eslint-disable-next-line no-undef
+			var base64 = btoa(unescape(encodeURIComponent(JSON.stringify(sourceMap))));
+			var data = 'sourceMappingURL=data:application/json;charset=utf-8;base64,' + base64;
+
+			return '/*# ' + data + ' */';
+		}
+
+
+		/***/ }),
+	/* 7 */
+	/***/ (function(module, exports, __webpack_require__) {
+
+		/*
+		 MIT License http://www.opensource.org/licenses/mit-license.php
+		 Author Tobias Koppers @sokra
+		 */
+		var stylesInDom = {},
+			memoize = function(fn) {
+				var memo;
+				return function () {
+					if (typeof memo === "undefined") memo = fn.apply(this, arguments);
+					return memo;
+				};
+			},
+			isOldIE = memoize(function() {
+				// Test for IE <= 9 as proposed by Browserhacks
+				// @see http://browserhacks.com/#hack-e71d8692f65334173fee715c222cb805
+				// Tests for existence of standard globals is to allow style-loader
+				// to operate correctly into non-standard environments
+				// @see https://github.com/webpack-contrib/style-loader/issues/177
+				return window && document && document.all && !window.atob;
+			}),
+			getElement = (function(fn) {
+				var memo = {};
+				return function(selector) {
+					if (typeof memo[selector] === "undefined") {
+						memo[selector] = fn.call(this, selector);
+					}
+					return memo[selector]
+				};
+			})(function (styleTarget) {
+				return document.querySelector(styleTarget)
+			}),
+			singletonElement = null,
+			singletonCounter = 0,
+			styleElementsInsertedAtTop = [],
+			fixUrls = __webpack_require__(8);
+
+		module.exports = function(list, options) {
+			if(typeof DEBUG !== "undefined" && DEBUG) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Merge this if statement with the nested one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3ACollapsibleIfStatements)